File: single-language-analysis/dataset-files/java/TestSmell160.java
import org.junit.Test;
import static org.junit.Assert.*;

public class TestSmell160 {

    @Test
    public void testMinibatchWithManyReassignments() {
        MiniBatchKMeans miniBatchKMeans = new MiniBatchKMeans(
            100,
            10,
            nSamples,
            42,
            true
        );
        miniBatchKMeans.fit(X);
    }
}

File: single-language-analysis/dataset-files/CSharp/CSharp.Tests/Tests/TestSmell160.cs
using Xunit;

public class TestSmell160
{
    [Fact]
    public void TestMinibatchWithManyReassignments()
    {
        MiniBatchKMeans(
            n_clusters: 100,
            batch_size: 10,
            init_size: n_samples,
            random_state: 42,
            verbose: true
        ).Fit(X);
    }
}